src/Build/Evaluation/Evaluator.cs

```diff
@@ -1785,7 +1785,14 @@ private List<ProjectRootElement> ExpandAndLoadImports(string directoryOfImportin
                 }
 
                 // Combine SDK path with the "project" relative path
-                sdkResult = _sdkResolverService.ResolveSdk(_submissionId, sdkReference, _evaluationLoggingContext, importElement.Location, solutionPath, projectPath, _interactive, _isRunningInVisualStudio);
+                try
+                {
+                    sdkResult = _sdkResolverService.ResolveSdk(_submissionId, sdkReference, _evaluationLoggingContext, importElement.Location, solutionPath, projectPath, _interactive, _isRunningInVisualStudio);
+                }
+                catch (Exception e)
```

Comment:
Is there a particular exception we expect this to throw?

Reply:
I was thinking of creating an `SdkResolverException` of some kind, but couldn't think of anything specific to add to it. In theory it could hold the resolver's name, sdk name, basically whatever's being used in this string.format. Then the evaluator would do exactly what's happening here.

Now that I'm actively thinking about it, having the evaluator check specifically for an sdkresolverexception might make more sense. If anything else throws (outside of the resolvers actually resolving) then we'd immediately be aware of a new case.

Reply:
Yeah, I don't think we should be throwing the base `Exception` class.  There may already be an exception type defined that would make sense here, so we might not need to create a new one.  @rainersigwald, any suggestiens?